src/Multilingual-Encodings.package/Unicode.class/class/isLetter..st

```diff
@@ -1,8 +1,7 @@
 character classification
-isLetter: char 
-	| value codeCat |
-	value := char charCode.
-	value > (GeneralCategory size - 1)
-		ifTrue: [^ false].
-	^ (codeCat := GeneralCategory at: value + 1) >= Ll
-		and: [codeCat <= Lu]
\ No newline at end of file
+isLetter: char
+	| index |
+	index := char charCode + 1.
+	index > GeneralCategory size
+		ifTrue: [ ^ false ].
+	^ (GeneralCategory at: index) between: Ll and: Lu
```

Comment:
https://en.wikipedia.org/wiki/Template:General_Category_(Unicode) shows five letter categories. Since it is not clear that Lu = Ll + 1 or even Ll+ 4, it would be well to document whether all five categories are encompassed and other assumptions about how Pharo implements the categories.

Reply:
This relies on the category initialization code, which sorts the category names alphabetically.  Hence, the original code is correct, although less clear than it might be.  It's also faster than checking each of the 5 individually.  I'm inclined to leave this be.